src/examples/checkbox/checkbox.tsx

```diff
@@ -27,6 +27,7 @@ export class CheckboxExample {
                         label="My fab checkbox"
                         id="fab"
                         checked={this.value}
+                        required={true}
```

Comment:
I'll make this switchable, just like `disabled`, if that's ok with you.

Reply:
Yeah, of course! :smile: